vnext/src/Libraries/Components/Button.windows.js

```diff
@@ -407,6 +427,7 @@ class Button extends React.Component<
           testID={testID}
           disabled={disabled}
           onPress={onPress}
+          tabIndex={tabIndex}
```

Comment:
this else block is for non-Windows platforms. is tabIndex a property on them, as well?

Reply:
Ah this shouldn't be needed I'll remove.